packages/web-app-files/src/services/folder/loaderSpace.ts

```diff
@@ -53,8 +53,9 @@ export class FolderLoaderSpace implements FolderLoader {
       try {
         store.commit('Files/CLEAR_CURRENT_FILES_LIST')
 
-        const resources = yield webdav.listFiles(space, { path, fileId })
-        let currentFolder = resources.shift()
+        const listFilesResult = yield webdav.listFiles(space, { path, fileId })
+        const resources = listFilesResult.children
+        let currentFolder = listFilesResult.folder
```

Comment:
Looking at this code, maybe `node` is still our best bet. But maybe keeping listFiles as a name wasn't too smart either. 
propfindResult.resource or similar seems okayish? Dunno... 

The result could be destructured directly

Reply:
yes right, but keeping listFiles as the method name?

Reply:
I would like to avoid the word `propfind`. 😉 `listFiles` as method name is ok IMO.

Reply:
Stop stop stop. Why not `listFiles` and `listFilesResult`?!

Reply:
The word `propfind` outside of the web-client feels wrong.